packages/shared/src/components/feeds/FeedSettings/sections/FeedSettingsAISection.tsx

```diff
@@ -51,22 +51,56 @@ export const FeedSettingsAISection = (): ReactElement => {
 
   return (
     <>
+      {!isPlus && (
+        <>
+          <div className="flex w-full items-center rounded-12 border border-border-subtlest-tertiary bg-action-plus-float p-3">
+            <Typography type={TypographyType.Callout}>
+              Upgrade and use daily.dev&apos;s AI Superpowers!
+            </Typography>
+            <Link href={plusUrl} passHref>
+              <Button
+                className="ml-auto w-fit"
+                tag="a"
+                type="button"
+                variant={ButtonVariant.Primary}
+                size={ButtonSize.Medium}
+                icon={<DevPlusIcon className="text-action-plus-default" />}
+                onClick={() => {
+                  logSubscriptionEvent({
+                    event_name: LogEvent.UpgradeSubscription,
+                    target_id: TargetId.FeedSettings,
```

Comment:
Just make sure to let Ido know about this chance (and removal of the others) 
They might need to know for event purposes